### PR TITLE
Add download.schneider-electric.com host target

### PR DESCRIPTION
**Pull request by wesinator**

### CODE REVIEW

File: src/chrome/content/rules/Schneider-Electric.com.xml [hunk 1]
```diff
@@ -71,5 +72,8 @@
 
 	<rule from="^http://(?:www\.)?schneider-electric\.com/(download/|navigation/|(?:product|solution)s(?:$|[?/])|resource/|site/(?:home|marketing_center)/(?:image|j)s/|site/tasks/sites/|stat/)"
 		to="https://www.schneider-electric.com/$1" />
```
Comment (J0WI) [inline src/chrome/content/rules/Schneider-Electric.com.xml]: Could you also remove the obsolete exclusions and the comments above?

Comment (wesinator) [reply src/chrome/content/rules/Schneider-Electric.com.xml]: shouldn't these updates be done under a separate PR ?

Comment (J0WI) [reply src/chrome/content/rules/Schneider-Electric.com.xml]: Would be great if you can fix it here. This kind of regex has some performance impact and we can also get rid of the whitelist entry.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**